src/main/java/racinggame/core/Car.java

```diff
@@ -1,26 +1,32 @@
-package step3.racinggame.core;
+package racinggame.core;
 
 public class Car {
 
     private static final int FORWARD_DISTANCE = 1;
     private static final int STANDSTILL = 0;
     private static final int FUEL_TO_MOVE = 4;
 
+    private String name;
```

Comment:
```suggestion
    private final String name;
```

이름은 변경되는 값이 아니니 `final`이 붙어도 되겠네요~

Reply:
@choihz 추가했습니다.